pages/thank-you.js

```diff
@@ -4,34 +4,33 @@ export default function Home() {
   return (
     <html>
       <head>
-        <title>{'Thank You'}</title>
+        <title>{'Thank You!'}</title>
       </head>
       <body>
         <div className="hero min-h-screen bg-base-200">
-          <div className="text-center hero-content">
+          <div className="hero-content">
             <div className="max-w-md">
-              <h1 className="mb-5 text-5xl font-bold">{'Thank You'}</h1>
-              <p>{'With gratitude from Margarita Humanitarian Foundation.'}</p>
+              <h1 className="mb-5 text-5xl font-bold">{'Thank You!'}</h1>
               <p>
                 {
-                  'Thanks so much for your generosity. We promise to use it to help the causes you indicated your support for, and to be extra thoughtful about putting it to good use.'
+                  'With gratitude from Margarita Humanitarian Foundation. Thanks so much for your generosity. We promise to use it to help the causes you indicated your support for, and to be extra thoughtful about putting it to good use.'
                 }
               </p>
-              <h2 className="m-5 text-3xl font-bold">{'Receipt to Follow'}</h2>
+              <h2 className="m-0 mt-10 mb-2 text-3xl font-bold">{'Receipt to Follow'}</h2>
```

Comment:
Please put a newline before the `{` and after the `}`
Alternatively run `eslint . --fix` if you have eslint installed.

Reply:
Thank you very much for the detailed comments, I have made the changes as suggested!